src/components/Icons/IconNext.tsx

```diff
@@ -1,6 +1,10 @@
 import { Icon } from '@aws-amplify/ui-react';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
This seems silly, but I can't figure our another way to solve the issue of duplicate IDs in this Nextjs logo. The `<linearGradient>` elements are used to provide the subtle gradient in the Nextjs logo, and those need to be referred to by id. However, if there are more than one of these elements in the DOM with the same id, one of the SVGs will not have the gradient applied (rendering it invisible).  Used uuid to generate a unique id instead. 🤷‍♀️